bioptim/limits/objective_functions.py

```diff
@@ -95,6 +95,20 @@ def print(self):
         """
         raise NotImplementedError("Printing of ObjectiveList is not ready yet")
 
+    @staticmethod
+    def get_nlp_objectives(nlp):
```

Comment:
same question as constraints (but probably less relevant here)

Reply:
Yes, maybe it seems less relevant in this case. I have to think about that when drawing the graph...

Reply:
Done